packages/main/src/types/StandardCalendarLegendItemsType.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Enum for standard calendar legend items.
+ *
+ * @readonly
+ * @enum {string}
+ * @public
+ * @author SAP SE
+ * @alias sap.ui.webc.main.types.CalendarLegendStandardItemsType
+ */
+enum CalendarLegendStandardItemsType {
```

Comment:
The name of the file does not patch the enum name StandardCalendarLegendItemsType vs CalendarLegendStandardItemsType. Rename everything to CalendarLegendItemType